OMEdit/OMEditGUI/Editors/MetaModelicaEditor.cpp

```diff
@@ -43,6 +43,12 @@ MetaModelicaEditor::MetaModelicaEditor(QWidget *pParent)
   mpPlainTextEdit->setCanHaveBreakpoints(true);
   /* set the document marker */
   mpDocumentMarker = new DocumentMarker(mpPlainTextEdit->document());
+  mpMetaModelicaHighlighter = new MetaModelicaHighlighter (OptionsDialog::instance()->getMetaModelicaEditorPage(),this->getPlainTextEdit());
```

Comment:
Why are you creating a new highlighter here?
We already have the highlighter in `ModelWidget` class.